styles.css

```diff
@@ -1273,6 +1273,14 @@ textarea::-webkit-input-placeholder {
     margin-right: 1vw;
   }
 
+  #error-label {
+    color: crimson;
+    margin-top: 20px;
+    font-size: 1.2rem;
+    margin-left: 30px;
+    padding: 0;
+  }
+
```

Comment:
- [ ] Excellent job adding validation 💯, but there is a minor detail, in the requirements says that the error message should be near the submit button. Kindly, relocate it 🦘

![image](https://user-images.githubusercontent.com/91301423/202459151-dba0711b-d844-4085-b9b5-9e7b34aba385.png)

Reply:
Fixed.